src/mscorlib/src/System/Reflection/StrongNameKeyPair.cs

```diff
@@ -49,6 +39,7 @@ public class StrongNameKeyPair : IDeserializationCallback, ISerializable
         private String  _keyPairContainer;
```

Comment:
Actually - scratch this.

Instead, could you please add the ctor that takes string; and add the PublicKey property and implement it by throwing PNSE exception.